CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ Classify the change according to the following categories:
 ### Fixed 
 - Added `export_rate_beyond_net_metering_limit` to list of inputs to be converted to type Real, to avoid MethodError if type is vector of Any. 
 - Fix blended CRB processing when one or more load types have zero annual energy
+- When calculating CHP fuel intercept and slope, use 1 for the HHV because CHP fuel measured in units of kWh, instead of using non-existent **CHP.fuel_higher_heating_value_kwh_per_gal**
```

Comment:
Where was this implemented?

Reply:
https://github.com/NREL/REopt.jl/blob/54dbb3f226f873f0b8ba415a3d43be1f856e3608/src/outagesim/backup_reliability.jl#L803